include/mm/regions.h

```diff
@@ -152,6 +152,8 @@ static inline paddr_t get_region_free_start(void *from) {
     return start;
 }
 
+bool in_reserved_range(paddr_t pa, size_t size);
```

Comment:
nit: please move it up to external declaractions section and prepend with `extern`